ops/toolbox/snippets/flatten-r43-archive.md

```
@@ -0,0 +1,27 @@
+<!-- TOOLBOX:START {"task":"Q20251008014500-arch","title":"Flatten redundant r43 archive nesting","summary":"Move redundant .bak backup out of nested archive and update cleanup records.","owner":"chatgpt","tags":["archives","cleanup"],"modes":["agent"],"created":"2025-10-08T03:46:34Z"} TOOLBOX:END -->
+
+# Flatten redundant r43 archive nesting
+
+## When to Use
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)